containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -97,9 +98,11 @@ def contentFilter(self):
 
 class HttpResponseDataFilteredPXE(HttpResponseDataFiltered):
     def contentFilter(self):
```

Comment:
Instead of initializing empty variables and appending content, I would suggest considering templating (if possible) in a style like this:

```
saltboot_template = """
...
  LABEL {{ entry_name }}
    MENU DEFAULT
    ONTIMEOUT {{ entry_name }}
...
"""
```

You can then use a `render_template` function to render the given content.

Alternatively, you could consider some form of object representation where the content would be parsed by a common engine.

This approach would be beneficial because in the current example, it's hard to determine the resulting content, especially with the presence of empty prefix spaces, special characters, and for loops over `splitlines()`.

Reply:
ok